rpcserver.go

```diff
@@ -2446,6 +2467,7 @@ func (r *rpcServer) marshalIssuanceProof(ctx context.Context,
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
I would consider moving the initialisation of `uniRoot` down to where its fields are populated
```
uniRoot.AssetName = assetLeaf.Asset.AssetGenesis.Name
uniRoot.Id = req.Id
```